truss-chains/truss_chains/model_skeleton.py

```diff
@@ -23,6 +24,24 @@ def __init__(
                 config["model_metadata"][definitions.TRUSS_CONFIG_CHAINS_KEY]
             )
         )
+
+        # Override chainlet_to_service if chainlet_service_config exists in extra config
```

Comment:
We should probably wrap this in a function and unit test this...and if there are larger-scoped integration tests, we should try to incorporate this behavior